app/javascript/dashboard/modules/contact/components/AddCustomAttribute.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <modal :show.sync="show" :on-close="onClose">
+    <woot-modal-header
+      :header-title="$t('CUSTOM_ATTRIBUTES.ADD.TITLE')"
+      :header-content="$t('CUSTOM_ATTRIBUTES.ADD.DESC')"
+    />
+    <form class="row" @submit.prevent="addCustomAttribute">
+      <woot-input
+        v-model.trim="attributeName"
+        :class="{ error: $v.attributeName.$error }"
+        class="medium-12 columns"
+        :error="$v.attributeName.$error"
```

Comment:
The `error`  prop should be the error message.

![Screenshot 2021-05-21 at 10 19 02 AM](https://user-images.githubusercontent.com/12408980/119083650-325b5e80-b9b5-11eb-8ac6-7e1033d4a7c7.png)